.eslintignore

```diff
@@ -0,0 +1,7 @@
+.circleci/*
+.githooks/*
+.idea/*
+.vscode/*
+node_modules/*
+./documentation/snippets
+./documentation/examples
```

Comment:
We definitely want these to get linted, we just want them to have their own rule overrides. I'm trying to understand, is the custom .eslintrc.js not working properly or something?